docs/guides/desktop/03-desktop-serve.md

```diff
@@ -44,12 +45,7 @@ project creation, but you can also hit the play button to start a project that's
 
  <img src="/images/dev-app-code-update.jpg" width="250" height="523" />
 
+6.  Continue making updates to your project to get familiar with this workflow. 
 
-#### Video Example
-
-<img src="/images/phonegap-app-developer-workflow-v2.gif" width="500" height="397"/>
+At this point you should check out [this guide](../develop/01-hello-app-explained.html) explaining important details about the default Hello PhoneGap application and mobile application development tips with PhoneGap in general.
```

Comment:
I think I can improve how we link URLs. Clean URLs are supported by DocPad, so we can reference `../develop/01-hello-app-explained` without the extension. I'll create an issue to look into this.